SignalBackgroundMerger.cpp

```diff
@@ -567,6 +568,9 @@ class SignalBackgroundMerger {
 
       HepMC3::GenEvent inevt;
       adapter->read_event(inevt);
+      if (signal){
```

Comment:
```suggestion
      if (signal && (signalFreq == 0.0)){
```
Right?

Reply:
Sure, we can do that! I just thought you wanted to allow it generally